media_cdn/snippets_test.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+#
+# Copyright 2022 Google, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for snippets."""
+
+import datetime
+
+import snippets
+
+
+EPOCH_TIME = 1650848400
+
+
+def test_sign_url(capsys):
```

Comment:
Nitpick, fix if you want: Those should be separate tests. So instead of 3 tests you maybe should have 8 in this file.

Reply:
No, please. At most I would re-org the code but won't change much :)